packages/x-engine/readme.md

```diff
@@ -119,6 +119,26 @@ export default const TeaserList = (props) => (
 );
 ```
 
+#### Client-side through n-ui
+In order to configure n-ui to compile x-dash based `.jsx` files you will also need to add the following to `n-ui-build.config.js`.
+
+```js
+// n-ui-build.config.js
+const xEngine = require('@financial-times/x-engine/src/webpack');
+
+module.exports = {
+	plugins: [
+		xEngine()
+	],
+	pragma: 'xEngine.h'
+};
+```
+
+You will also need to import `x-engine` at the top of each `.jsx` file.
+```javascript
+import xEngine from 'x-engine'
```

Comment:
xEngine doesn't have a default export so I believe this should be `* as xEngine` or use named imports such as `import { h } from '@financial-times/x-engine'`

Reply:
Confirmed: could we change this to advise using named imports?

Reply:
Will do this, though n-ui makes it work anyway.